README.md

```diff
@@ -2,12 +2,14 @@
 
 Cryptographic validation of Scuttlebutt messages in the form of Rust bindings for Node.js and WebAssembly.
 
-Perform batch verification and validation of SSB messages using [ssb-verify-signatures](https://crates.io/crates/ssb-verify-signatures) and [ssb-validate](https://github.com/mycognosist/ssb-validate) from the [Sunrise Choir](https://github.com/sunrise-choir).
+Perform batch verification and validation of SSB message values using [ssb-verify-signatures](https://crates.io/crates/ssb-verify-signatures) and [ssb-validate](https://github.com/mycognosist/ssb-validate) from the [Sunrise Choir](https://github.com/sunrise-choir).
 
 This package wraps [ssb-validate2-rsjs-node](https://github.com/ssb-ngi-pointer/ssb-validate2-rsjs-node) and [ssb-validate2-rsjs-wasm](https://github.com/ssb-ngi-pointer/ssb-validate2-rsjs-wasm) so to support Node.js (by compiling a native add-on) and Browsers (by compiling to WASM).
 
 ## Usage
 
+Usage differs slightly based on context: in browser (`WASM`) or on native (`node`). Please consult the [usage example in ssb-validate2-rsjs-wasm](https://github.com/ssb-ngi-pointer/ssb-validate2-rsjs-wasm#usage) for additional details.
```

Comment:
How does the usage differ?

Reply:
This is in relation to the `.ready()` initialisation requirement for `-wasm`. But of course we added that to the API for `-node` as well so maybe it's better to remove the "Usage differs..." section and rather update the usage example to include `ready()`?

Reply:
Yes! We want the APIs of these two to match perfectly. The person using `-node` doesn't need to know that `ready()` is unnecessary

Reply:
Just committed a change for this. Have I demonstrated the `ready()` step in the correct way?